websites/A/Anizium/presence.ts

```diff
@@ -0,0 +1,114 @@
+interface Video {
+	paused: boolean;
+	duration: number;
+	currentTime: number;
+}
+
+const presence = new Presence({
+		clientId: "1094616557763710976",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+	});
+
+let video: Video;
+
+presence.on("iFrameData", (msg: Video) => {
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Z3G1Ubm.png",
+		},
+		haberAd: string =
+			document
+				.querySelector(
+					"body > div.haberler > div > div > div:nth-child(7) > div > div > div.kart-header > h2"
+				)
+				?.textContent.trim() || null,
+		haberDetay: string =
+			document
+				.querySelector(
+					"body > div.haberler > div > div > div:nth-child(7) > div > div > div.kart-body > p:nth-child(4)"
+				)
+				?.textContent.trim() || null,
+		animeInfo: string =
+			document
+				.querySelector(
+					"body > div.container-fluid > div > div > div:nth-child(2) > div:nth-child(1) > span"
+				)
+				?.textContent.trim() || null,
+		profilAd: string =
+			document
+				.querySelector(
+					"body > div.container-fluid > div.row > div.col-md-10 > div:nth-child(6) > div:nth-child(1) > label"
+				)
+				?.textContent.trim() || null;
+
+	if (
+		window.location.pathname === "/" ||
```

Comment:
Use `document.location`. Also, destructure the properties (`pathname`, `href`)

Reply:
i did ty